[2022-10-14 14:28:54.228] [remoteagent] [info] Extension host agent started.
[2022-10-14 14:28:54.545] [remoteagent] [info] [127.0.0.1][1cf71700][ManagementConnection] New connection established.
[2022-10-14 14:28:54.609] [remoteagent] [info] [127.0.0.1][249f35b4][ExtensionHostConnection] New connection established.
[2022-10-14 14:28:54.628] [remoteagent] [info] [127.0.0.1][249f35b4][ExtensionHostConnection] <1512> Launched Extension Host Process.
[2022-10-14 14:28:56.471] [remoteagent] [error] ["CodeExpectedError: Could not find pty on pty host","    at a._throwIfNoPty (/root/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:27:7252)","    at a.updateTitle (/root/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:27:2984)","    at Object.call (/root/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:16:8386)","    at c.onPromise (/root/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:15:5833)","    at c.onRawMessage (/root/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:15:5216)","    at /root/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:15:4502","    at m.invoke (/root/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at D.deliver (/root/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at O.fire (/root/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.X (/root/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:9:20837)","    at process.emit (node:events:526:28)","    at emit (node:internal/child_process:938:14)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-10-14 14:29:00.145] [remoteagent] [error] [File Watcher (parcel)] Inotify limit reached (ENOSPC) (path: /)
[2022-10-14 15:34:31.835] [remoteagent] [warning] No ptyHost heartbeat after 6 seconds
[2022-10-14 15:41:16.359] [remoteagent] [info] [127.0.0.1][e13da5a5][ManagementConnection] New connection established.
[2022-10-14 15:41:16.376] [remoteagent] [info] [127.0.0.1][575454d3][ExtensionHostConnection] New connection established.
[2022-10-14 15:41:16.398] [remoteagent] [info] [127.0.0.1][575454d3][ExtensionHostConnection] <5108> Launched Extension Host Process.
[2022-10-14 15:41:18.709] [remoteagent] [error] ["CodeExpectedError: Could not find pty on pty host","    at a._throwIfNoPty (/root/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:27:7252)","    at a.updateTitle (/root/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:27:2984)","    at Object.call (/root/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:16:8386)","    at c.onPromise (/root/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:15:5833)","    at c.onRawMessage (/root/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:15:5216)","    at /root/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:15:4502","    at m.invoke (/root/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at D.deliver (/root/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at O.fire (/root/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.X (/root/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:9:20837)","    at process.emit (node:events:526:28)","    at emit (node:internal/child_process:938:14)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-10-14 15:41:19.202] [remoteagent] [error] [File Watcher (parcel)] Inotify limit reached (ENOSPC) (path: /)
[2022-10-14 17:25:26.617] [remoteagent] [info] [127.0.0.1][a93fff0b][ManagementConnection] New connection established.
[2022-10-14 17:25:26.635] [remoteagent] [info] [127.0.0.1][cc2a0004][ExtensionHostConnection] New connection established.
[2022-10-14 17:25:26.702] [remoteagent] [info] [127.0.0.1][cc2a0004][ExtensionHostConnection] <12402> Launched Extension Host Process.
[2022-10-14 17:25:29.330] [remoteagent] [error] ["CodeExpectedError: Could not find pty on pty host","    at a._throwIfNoPty (/root/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:27:7252)","    at a.updateTitle (/root/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:27:2984)","    at Object.call (/root/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:16:8386)","    at c.onPromise (/root/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:15:5833)","    at c.onRawMessage (/root/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:15:5216)","    at /root/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:15:4502","    at m.invoke (/root/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:11:145)","    at D.deliver (/root/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:11:2275)","    at O.fire (/root/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:11:1853)","    at process.X (/root/.vscode-server/bin/d045a5eda657f4d7b676dedbfa7aab8207f8a075/out/vs/platform/terminal/node/ptyHostMain.js:9:20837)","    at process.emit (node:events:526:28)","    at emit (node:internal/child_process:938:14)","    at processTicksAndRejections (node:internal/process/task_queues:84:21)"]
[2022-10-14 17:25:30.710] [remoteagent] [info] [127.0.0.1][a93fff0b][ManagementConnection] The client has disconnected gracefully, so the connection will be disposed.
[2022-10-14 17:25:30.725] [remoteagent] [info] [127.0.0.1][cc2a0004][ExtensionHostConnection] <12402> Extension Host Process exited with code: 0, signal: null.
[2022-10-14 17:25:37.098] [remoteagent] [info] [127.0.0.1][df8b4da0][ManagementConnection] New connection established.
[2022-10-14 17:25:37.102] [remoteagent] [info] [127.0.0.1][26a3d9d7][ExtensionHostConnection] New connection established.
[2022-10-14 17:25:37.111] [remoteagent] [info] [127.0.0.1][26a3d9d7][ExtensionHostConnection] <12546> Launched Extension Host Process.
[2022-10-14 17:25:41.700] [remoteagent] [error] [File Watcher (parcel)] Unexpected error: inotify_add_watch on '//proc/1486/task/12575/fdinfo' failed: No such file or directory (EUNKNOWN) (path: /)
[2022-10-14 17:25:41.701] [remoteagent] [error] [File Watcher (universal)] restarting watcher after error: inotify_add_watch on '//proc/1486/task/12575/fdinfo' failed: No such file or directory
[2022-10-14 17:25:42.928] [remoteagent] [error] [File Watcher (parcel)] Inotify limit reached (ENOSPC) (path: /)
[2022-10-14 17:30:27.288] [remoteagent] [info] [127.0.0.1][249f35b4][ExtensionHostConnection] <1512> Extension Host Process exited with code: 0, signal: null.
[2022-10-14 17:30:27.322] [remoteagent] [info] [127.0.0.1][575454d3][ExtensionHostConnection] <5108> Extension Host Process exited with code: 0, signal: null.
